etc/PlayOnLinux.desktop

```diff
@@ -1,9 +1,8 @@
 [Desktop Entry]
 Version=1.0
-Encoding=UTF-8
 Name=PlayOnLinux
-Comment=PlayOnLinux
-Type=Application %F
```

Comment:
I was done changes on behalf of desktop-file-validate utility and for this line I get following output:

`playonlinx.desktop: error: value "Application %F" for key "Type" in group "Desktop Entry" is not a registered type value ("Application", "Link" and "Directory")`

I'm not quite expert on *.desktop files so it's possible I have fixed it wrong.